src/jdk.jdwp.agent/windows/native/libdt_filesocket/fileSocketTransport_md.c

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2018, 2024 SAP SE. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#include <stdlib.h>
+#include <assert.h>
+
+#include "jni.h"
+#include "jvm.h"
+#include "fileSocketTransport.h"
+
+
+#include <windows.h>
+#include <winsock2.h>
+#include <afunix.h>
+
+/* Make sure winsock is initialized on Windows */
+BOOL WINAPI DllMain(HINSTANCE hinst, DWORD reason, LPVOID reserved)
+{
+    WSADATA wsadata;
+
+    if ((reason == DLL_PROCESS_ATTACH) && (WSAStartup(MAKEWORD(2, 2), &wsadata) != 0)) {
+        return JNI_FALSE;
+    }
+    else if (reason == DLL_PROCESS_DETACH) {
```

Comment:
Same here, formatting `} else if `

Reply:
Removed the else, since it is not needed at all.